ui/app/styles/components/toggle.scss

```diff
@@ -10,6 +10,16 @@ $size: 12px;
     cursor: not-allowed;
   }
 
+  &.is-flex {
+    display: flex;
+    align-items: center;
+
+    .label {
+      padding-left: 0.5rem;
+      padding-bottom: 1px;
+    }
+  }
+
```

Comment:
I’m not sure how acceptable this is… there’s nowhere to really target this toggle specifically but maybe it needs to be its own component Sass file 😞

Reply:
Yeah, there isn't a current better alternative. Ideally these types of variants/exceptions get cleaned up more systemically when we rip out Bulma.